src/mrpro/data/_KData.py

```diff
@@ -149,18 +139,25 @@ def from_file(
         num_total_unique = torch.as_tensor([len(unique_idxs[label]) for label in KDIM_SORT_LABELS]).prod()
 
         # Define function to find index label combinations. This is used to determine the dimensions of k1 and k2.
-        def idx_label_combination(average_idx, slice_idx, contrast_idx, phase_idx, repetition_idx, set_idx):
+        def idx_label_combination(
+            average_idx: list[int],
```

Comment:
these typhintes are wrong!
why doenst my my complain in line 160?
this function is called there with ints as paramters.

Reply:
will be refactored in #207 anyway